documentation/docs/library-get-started/basic-operations.md

```diff
@@ -0,0 +1,159 @@
+---
+sidebar_label: 'Basic Operations'
+sidebar_position: 3
+---
+
+# Basic Operations
+
+## Create Identifier
+
+Create a new file in **`src/create-identifier.ts`**, and add the following code.
+**Note**: in this guide we use did:ethr as the did method, you can use other method by declaring the provider as the parameter of didManagerCreate function
+
+```typescript
+import { agent } from './vckit/setup.js'
+
+async function main() {
+  const identifier = await agent.didManagerCreate({ alias: 'default' })
+  console.log(`New identifier created`)
+  console.log(JSON.stringify(identifier, null, 2))
+}
+
+main().catch(console.log)
+```
+Run this command to create identifier
+```bash
+node --loader ts-node/esm ./src/create-identifier.ts  
+```
+## Get the list of indentifiers
+
+Create a new file in **`src/list-indentifiers.ts`**, and add the following code.
+
+```typescript
+import { agent } from './vckit/setup.js'
+
+async function main() {
+  const identifiers = await agent.didManagerFind()
+
+  console.log(`There are ${identifiers.length} identifiers`)
+
+  if (identifiers.length > 0) {
+    identifiers.map((id) => {
+      console.log(id)
+      console.log('..................')
+    })
+  }
+}
+
+main().catch(console.log)
+```
+
+Run this command to get list of identifiers
+```bash
+node --loader ts-node/esm ./src/list-identifiers.ts  
```

Comment:
I am curious about this command. This project is a monorepo, so when I am in root and use this cmd. How can it work? Or should I setup a file with name list-identifiers.ts somewhere?

Reply:
yup. you need to create a file which contains the logic of getting the list of identifiers and name it 'list-identifiers.ts'. Is the document not clear enough?